components/Cards/SchoolCard/SchoolCard.js

```diff
@@ -50,6 +50,7 @@ class SchoolCard extends Component {
             cardFlipCallback={this.showFrontOfCard}
             locations={props.locations}
             logoSource={props.logoSource}
+            alt={`${props.schoolName} logo`}
```

Comment:
what if you generated the alt in the backschoolcard, but just passed school name here instead?

It's a bit of a disconnect for me to have alt defined here, when I don't even know about there being an image from here (in the code).